deploy/codebase/updateinit.js

```diff
@@ -35,6 +35,33 @@ function installBarcodeLib(cb) {
   command(cmd, cb);
 }
 
+function installDeviceConfig (cb) {
+  try {
+    const currentDeviceConfigPath = '/opt/apps/machine/lamassu-machine'
+    const newDeviceConfigPath = '/tmp/extract/package/subpackage/hardware/' + hardwareCode + '/device_config.json'
+    const currentDeviceConfig = require(currentDeviceConfigPath)
+    const newDeviceConfig = require(newDeviceConfigPath)
+
+    if (currentDeviceConfig.billDispenser) {
+      newDeviceConfig.billDispenser.mode = currentDeviceConfig.billDispenser.model
+      newDeviceConfig.billDispenser.device = currentDeviceConfig.billDispenser.device
+    }
+    if (currentDeviceConfig.billValidator) {
+      newDeviceConfig.billValidator.rs232.device = currentDeviceConfig.billValidator.rs232.device
+    }
+
+    // Pretty-printing the new configuration to retain its usual form.
+    const adjustedDeviceConfig = JSON.stringify(newDeviceConfig, null, 2)
+    fs.writeFileSync(newDeviceConfigPath, adjustedDeviceConfig)
+
+    const cmd = 'cp /tmp/extract/package/subpackage/hardware/' + hardwareCode + '/device_config.json /opt/apps/machine/lamassu-machine'
```

Comment:
Why not use ``newDeviceConfigPath`` and ``currentDeviceConfigPath`` here? Why use ``cp`` when we can just directly write to ``currentDeviceConfigPath``?

Reply:
You're it's redundant. Removed and updated `writeFileSync` accordingly.